.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: java
+jdk:
+  - oraclejdk8
+addons:
+  postgresql: '9.4'
+  mariadb: '10.1'
+  apt:
+    packages:
+      - oracle-java8-installer
+#  artifacts:
+#    paths:
+#      - $(find $HOME -name surefire-reports | tr "\n" ":")
+#    debug: true
+#  s3_region: 'us-west-2'
+cache:
+  directories:
+    - $HOME/.m2
+env:
+  - DB=h2
+  - DB=pgsql
+  - DB=mariadb
+install:
+  - mvn -v
+  - mvn -s settings-example.xml -B -q clean compile -DskipTests=true
+before_script: 
+  - if [[ "$DB" == "pgsql" ]]; then psql -U postgres -f 'travis/pgsql.sql';fi 
+  - if [[ "$DB" == "mariadb" ]]; then mysql -u root < 'travis/mariadb.sql'; fi
+script:
+  - if [[ "$DB" == "h2" ]]; then mvn -s settings-example.xml -Ph2 test;fi 
```

Comment:
How comes you do the tests after compile? Seems btw. it'd miss the integration tests that way?